sqlserver/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 * [Fixed] Always cast provided port to str. See [#13055](https://github.com/DataDog/integrations-core/pull/13055).
 * [Removed] Remove sqlserver tag truncation for metrics. See [#13211](https://github.com/DataDog/integrations-core/pull/13211).
 
-## 9.0.2 / 2022-10-14
+## 9.0.2 / 2022-10-14 / Agent 7.40.0
```

Comment:
I think we've missing a reference to https://github.com/DataDog/integrations-core/pull/13089 in the 7.40.0 release. It looks like it was cherry-picked and is on the 7.40.0 branch: https://github.com/DataDog/integrations-core/blob/7.40.x/sqlserver/datadog_checks/sqlserver/statements.py

Reply:
https://github.com/DataDog/integrations-core/pull/13123 is the backport of the https://github.com/DataDog/integrations-core/pull/13089 PR which was merged into master. https://github.com/DataDog/integrations-core/pull/13123 is included in the 7.40.0 changelog so it should be OK